src/data/battle-anims.ts

```diff
@@ -1428,7 +1428,9 @@ export class MoveAnim extends BattleAnim {
   public move: Moves;
 
   constructor(move: Moves, user: Pokemon, target: BattlerIndex, playOnEmptyField = false) {
-    super(user, globalScene.getField()[target], playOnEmptyField);
+    /** Set target to the user pokemon as a dummy if no target is found to avoid crashes */
+    const dummyPokemon = user;
+    super(user, globalScene.getField()[target] ?? dummyPokemon, playOnEmptyField);
```

Comment:
```suggestion
    // Set target to the user pokemon if no target is found to avoid crashes
    super(user, globalScene.getField()[target] ?? user, playOnEmptyField);
```
Since all you're doing is assigning `user` to the `dummyPokemon` variable you can just use `user` directly instead for the same result.
Same with the other time in `move-effect-phase.ts`.